src/dex-ui/services/DexService/dao/DAOService.ts

```diff
@@ -91,36 +91,58 @@ async function fetchMultiSigDAOs(eventTypes?: string[]): Promise<MultiSigDAODeta
     new ethers.utils.Interface(MultiSigDAOFactoryJSON.abi),
     eventTypes
   );
-
-  return Promise.all([
-    ...logs.map(async (log): Promise<MultiSigDAODetails> => {
-      const argsWithName = getEventArgumentsByName<MultiSigDAOCreatedEventArgs>(log.args, ["owners", "webLinks"]);
-      const { daoAddress, safeAddress, inputs } = argsWithName;
-      const { admin, isPrivate, threshold: _threshold, title } = inputs;
-      const safeLogs = await fetchHederaGnosisSafeLogs(safeAddress);
-      const owners = getOwners(safeLogs);
-      const threshold = getThreshold(safeLogs, _threshold);
-
-      const { name, description, logoUrl, webLinks } = await fetchMultiSigDAOSettingsPageDetails(
-        AccountId.fromSolidityAddress(daoAddress).toString(),
-        [DAOEvents.DAOInfoUpdated]
-      );
+  const multiSigEventResults = logs.map(async (log): Promise<MultiSigDAODetails> => {
+    const argsWithName = getEventArgumentsByName<MultiSigDAOCreatedEventArgs>(log.args, ["owners", "webLinks"]);
+    const { daoAddress, safeAddress, inputs } = argsWithName;
+    const { admin, isPrivate, threshold: _threshold, title } = inputs;
+    const safeLogs = await fetchHederaGnosisSafeLogs(safeAddress);
+    const owners = getOwners(safeLogs);
+    const threshold = getThreshold(safeLogs, _threshold);
+
+    /** START - TODO: Need to apply a proper fix */
+    let accountId;
+    try {
+      accountId = AccountId.fromSolidityAddress(daoAddress).toString();
+    } catch (e) {
+      console.log(daoAddress, e /* AccountId.fromSolidityAddress(daoAddress).toString() */);
       return {
         type: DAOType.MultiSig,
-        accountId: AccountId.fromSolidityAddress(daoAddress).toString(),
+        accountId: "Error",
         adminId: AccountId.fromSolidityAddress(admin).toString(),
-        name,
-        logoUrl,
-        title,
-        description,
-        isPrivate,
-        webLinks,
+        name: "Error",
+        logoUrl: "Error",
+        title: "Error",
+        description: "Error",
+        isPrivate: false,
+        webLinks: ["Error"],
         safeId: AccountId.fromSolidityAddress(safeAddress).toString(),
         ownerIds: owners.map((owner) => AccountId.fromSolidityAddress(owner).toString()),
         threshold,
       };
-    }),
-  ]);
+    }
+    /** END - TODO: Need to apply a proper fix */
```

Comment:
Temporary emergency fix. Parsing some DAO Account IDs with `AccountId.fromSolidityAddress` results in a "Negative numbers are not allowed in IDs" error.